anchor_check.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+#### GUIDE ###############################################
+# download this script with wget LINKTOFILE
+# assumes lncli is globally available: whereis lncli
+# execute with bash anchor_check.sh
```

Comment:
Am I assuming correct that for Umbrel, you gotta start the script like so: `bash anchor_check.sh umbrel` ?

Reply:
Yes, good point. I'll add this to the Guide section